recipes-graphics/wayland/weston-init/weston-waitforcard.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+DEFAULT_CARD=/dev/dri/card0
```

Comment:
However, I prefer we still keep the possibility of using FBDev, since it is useful in some cases.